conda/install.py

```diff
@@ -497,26 +507,28 @@ def is_linked(prefix, dist):
     except IOError:
         return None
 
-def delete_trash(prefix):
+def delete_trash():
```

Comment:
Huh, I don't know why that argument was put there. It may need to be kept for backwards compatibility for at least a version because conda-build uses this function.